scripts/setup/macos-10.15/install_cbmc.sh

```diff
@@ -4,5 +4,15 @@
 
 set -eux
 
-# Install CBMC for macOS 10.15
-brew install cbmc
+if [[ $# -ne 1 ]]; then
+  echo "$0: Error: Specify the CBMC version file"
+  exit 1
+fi
+
+CBMC_VERSION_FILE=$1
```

Comment:
We can change this later, but here are my two cents. We should create one file: kani-dependencies. The format should be:
```
CBMC_VERSION="5.6"
CBMC_VIEWER_VERSION="3.2"
...
```
You can now use `source kani-dependencies` inside your script and those variables are available in your script. Note that you can also use the same file inside python scripts and it is also a valid toml format.

Reply:
Done.